config/sync/views.view.stanford_opportunities.yml

```diff
@@ -0,0 +1,3947 @@
+    display_title: 'Filtered List'
+    display_plugin: viewfield_block
+    position: 1
+    display_options:
+      display_extenders: {  }
+      display_description: ''
+      block_description: ''
+      block_hide_empty: false
+      block_category: 'Lists (Views)'
+      defaults:
+        fields: true
+        style: false
+        row: false
+        arguments: false
+        filters: false
+        filter_groups: false
+        pager: false
+        empty: false
+        css_class: false
+        title: false
+        link_display: false
+        link_url: false
+      style:
+        type: default
+        options:
+          row_class: flex-md-12-of-12
+          default_row_class: 1
+          uses_fields: 0
+      row:
+        type: 'entity:node'
+        options:
+          relationship: none
+          view_mode: teaser
+      arguments: {  }
+      exposed_block: false
+      filters:
+        status:
+          id: status
+          table: node_field_data
+          field: status
+          entity_type: node
+          entity_field: status
+          plugin_id: boolean
+          value: '1'
+          group: 1
+          expose:
+            operator: ''
+            operator_limit_selection: false
+            operator_list: {  }
+        type:
+          id: type
+          table: node_field_data
+          field: type
+          relationship: none
+          group_type: group
+          admin_label: ''
+          entity_type: node
+          entity_field: type
+          plugin_id: bundle
+          operator: in
+          value:
+            stanford_opportunity: stanford_opportunity
+          group: 1
+          exposed: false
+          expose:
+            operator_id: ''
+            label: ''
+            description: ''
+            use_operator: false
+            operator: ''
+            operator_limit_selection: false
+            operator_list: {  }
+            identifier: ''
+            required: false
+            remember: false
+            multiple: false
+            remember_roles:
+              authenticated: authenticated
+            reduce: false
+          is_grouped: false
+          group_info:
+            label: ''
+            description: ''
+            identifier: ''
+            optional: true
+            widget: select
+            multiple: false
+            remember: false
+            default_group: All
+            default_group_multiple: {  }
+            group_items: {  }
+        term_node_tid_depth:
+          id: term_node_tid_depth
+          table: node_field_data
+          field: term_node_tid_depth
+          relationship: none
+          group_type: group
+          admin_label: ''
+          operator: or
+          value:
+            79: '79'
+            81: '81'
+            82: '82'
+            80: '80'
+          group: 1
+          exposed: true
+          expose:
+            operator_id: term_node_tid_depth_op
+            label: 'Filter by Quarter'
```

Comment:
We can't hard wire these labels. We'll have to find or build a way to do these filters in the same way that we have the authoring widget. For now, lets remove the filtering blocks.